packages/e2e-tests/specs/editor/blocks/navigation.test.js

```diff
@@ -54,9 +54,13 @@ async function updateActiveNavigationLink( { url, label } ) {
 		await page.waitForXPath( `//span[@class="block-editor-link-control__search-item-title"]/mark[text()="${ url }"]` );
 		await page.keyboard.press( 'Enter' );
 	}
+
 	if ( label ) {
 		await page.click( '.wp-block-navigation-link__content.is-selected' );
+		const textContent = await page.evaluate( () => document.activeElement.textContent );
+		expect( textContent ).toBe( url );
 		await pressKeyWithModifier( 'primary', 'a' );
```

Comment:
No idea why, but the second time around the test fails to delete the text. 🤔 

This is only happening on travis, locally they pass everytime. 😢